zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/ImageServicePublishingResult.java

```diff
@@ -0,0 +1,44 @@
+package com.github.onsdigital.zebedee.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ImageServicePublishingResult {
+    private int totalImages;
+    private List<ImageDetails> unpublishedImages = new ArrayList<>();
+
+    public ImageServicePublishingResult(int totalImages) {
+        this.totalImages = totalImages;
+    }
+
+    public int getTotalImages() {
+        return totalImages;
+    }
+
+    public List<ImageDetails> getUnpublishedImages() {
+        return unpublishedImages;
+    }
+
+    public void addUnpublishedImage(String id, String status) {
+        unpublishedImages.add(new ImageDetails(id, status));
```

Comment:
Ummmm, yes AND no? Yes, if I'm writing properly robust Java then of course, but given we're adding a simple function to already complicated code in a deprecated service I went down the 'YAGNI' route knowing the only thing calling this would have values. I should probably add Exceptiony gubbins but given how sensitive the exception handling in this area is I was nervous. Happy to take advice.